src/components/FilterMenu/FilterMenu.vue

```diff
@@ -180,6 +180,9 @@ function getRelativeTimeString(date: Date): string {
 
 const updatedAt = ref();
 const { resume, pause } = useIntervalFn(() => {
+  if (!spacesStore.updatedAt)
+    return;
+
```

Comment:
Unrelated change, sorry! Small fix to prevent a race condition, I've seen this being undefined during development.